WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -137,10 +137,14 @@ - (void)toggleLikedForPost:(ReaderPost *)post success:(void (^)())success failur
         // Define success block. Make sure work is performed on the main queue
         void (^successBlock)() = ^void() {
             dispatch_async(dispatch_get_main_queue(), ^{
+                NSDictionary *properties = @{
+                                              WPAppAnalyticsKeyPostID: readerPost.postID,
+                                              WPAppAnalyticsKeyBlogID: readerPost.siteID
+                                              };
                 if (like) {
-                    [WPAppAnalytics track:WPAnalyticsStatReaderArticleLiked withBlogID:readerPost.siteID];
+                    [WPAnalytics track:WPAnalyticsStatReaderArticleLiked withProperties:properties];
```

Comment:
Actually can you switch this to use the Track method on `WPAppAnalytics` - @daniloercoli added convenience methods that pass through the event tracking to WPAnalytics. Another layer but it at least makes all of the tracking calls go through `WPAppAnalytics` app-wide.